OptimizelySDKCore/OptimizelySDKCore/OPTLYLoggerMessages.h

```diff
@@ -26,7 +26,16 @@ extern NSString *const OPTLYLoggerMessagesVariationUserAssigned;
 // info
 extern NSString *const OPTLYLoggerMessagesActivationSuccess;
 extern NSString *const OPTLYLoggerMessagesConversionSuccess;
+extern NSString *const OPTLYLoggerMessagesConversionFailure;
 // error
+extern NSString *const OPTLYLoggerMessagesUserIdInvalid;
```

Comment:
We just refactored the code, why these constants are added, any log or case were missing in the SDK.?